pkg/selinux/setroubleshoot-view.jsx

```diff
@@ -427,7 +427,7 @@ export class SETroubleshootPage extends React.Component {
                 shell={ "semanage import <<EOF\n" + this.props.selinuxStatus.shell.trim() + "\nEOF" }
                 ansible={ this.props.selinuxStatus.ansible }
                 entries={ this.props.selinuxStatus.modifications }
-                failed={ this.props.selinuxStatus.failed }
+                failed={this.props.selinuxStatus.failed ? _("Error running semanage to discover system modifications") : null}
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19370-20230922-100423-663bbd9c-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)